src/Hedgehog/Range.fs

```diff
@@ -102,59 +103,44 @@ module Range =
         // sufficiently accessible.
 
         /// Truncate a value so it stays within some range.
-        let clamp (x : 'a) (y : 'a) (n : 'a) : 'a =
+        let clamp (x : 'a) (y : 'a) (bound : 'a) : 'a =
```

Comment:
should `x` and `y` be `minimum` and `maximum`?

Reply:
I don't think so; see the new commit.